cetmix_tower_server/security/cx_tower_variable_value_security.xml

```diff
@@ -13,6 +17,59 @@
 
     </record>
 
+    <record id="cx_tower_variable_value_action_rule_group_user_access" model="ir.rule">
+        <field
+            name="name"
+        >Tower variable value: user access to variable values in plan line action
+            rule</field>
+        <field name="model_id" ref="model_cx_tower_variable_value" />
+        <field name="domain_force">
+            [
+            ('plan_line_action_id.access_level', '=', '1')
```

Comment:
What if access level == 1, but this plan is linked to specific servers and current used doesn't have access to them?

Reply:
I've updated the rules